apps/cowswap-frontend/src/modules/appData/updater/AppDataInfoUpdater.tsx

```diff
@@ -21,10 +22,13 @@ export const AppDataUpdater = React.memo(({ slippage, orderClass }: AppDataUpdat
 
   const slippageBips = percentToBips(slippage)
   const utm = useUtm()
+  const hooks = useAppDataHooks()
```

Comment:
Does it make sense to store the hooks independently and load them into the appData like this?